Dockerfile

```diff
@@ -1,18 +1,23 @@
 FROM alpine:edge
 
+ENV K8S_VERSION=v1.6.3
```

Comment:
Considering the goal of this image, should we just use latest instead? Do we care about the version at all, as long as it's recent?